index.html

```diff
@@ -284,6 +286,38 @@ <h2>Контакты</h2>
 
     <p class="htmlacademy-copyright">Разработано —<br><a href="http://htmlacademy.ru">htmlacademy.ru</a></p>
   </footer>
+  <section class="modal-write-us">
+    <h2 class="visually-hidden">Напишите нам!</h2>
+    <form action="https://echo.htmlacademy.ru" method="post">
+      <ul>
+        <li class="modal-write-us-name">
+          <p>Ваше имя:
+            <input type="text" name="name" id="name-place" placeholder="Имя Фамилия">
+            <label class="visually-hidden" for="name-place">Имя и фамилия</label></p>
+        </li>
+        <li class="modal-write-us-email">
+          <p>Ваше e-mail:
+            <input type="text" name="email" id="email-place" placeholder="email@example.com">
```

Comment:
```suggestion
            <input type="email" name="email" id="email-place" placeholder="email@example.com">
```
Типы инпутов: http://htmlbook.ru/html/input/type